sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/BlobDecryptionPolicy.java

```diff
@@ -104,6 +104,7 @@ public Mono<HttpResponse> process(HttpPipelineCallContext context, HttpPipelineN
                      * If there was no range set on the request, we skipped instantiating a BlobRange as we did not have
                      * encryption data at the time. Instantiate now with a BlobRange that indicates a full blob.
                      */
+                    // when creating the encrypted range, pass in the new value
```

Comment:
Let's also start to remove some of these comments that were added to help walk through the implementation as we get this out of a draft PR.